src/main/scala/com/codacy/api/client/CodacyClient.scala

```diff
@@ -25,7 +26,7 @@ class CodacyClient(
     apiToken.map(t => "api_token" -> t) ++
     projectToken.map(t => "project_token" -> t)
 
-  private val remoteUrl = apiUrl.getOrElse("https://api.codacy.com") + "/2.0"
+  private val remoteUrl = new URL(new URL(apiUrl.getOrElse("https://api.codacy.com")), "/2.0").toString()
```

Comment:
@lolgab, perhaps you have a suggestion on how to do this in a more "Scala way"? :smile: 

We should also have some error handling here in case the URL coming from user input is malformed. :thinking: Can you help?

Reply:
I guess this is fine.. I was thinking about files, that's why I told you there was a "Scala way" to interpolate 😄 
I'm merging this one.